topic-operator/pom.xml

```diff
@@ -221,6 +225,36 @@
             <groupId>io.vertx</groupId>
             <artifactId>vertx-micrometer-metrics</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.strimzi</groupId>
+            <artifactId>operator-common</artifactId>
+            <classifier>tests</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this needed?

Reply:
I need this one as I use MockCertManager in ReplicasChangeClientTest.